week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/main.ts

```diff
@@ -0,0 +1,5 @@
+import { root } from './shop/root.component'
+
+const $app: HTMLElement = document.querySelector('#app') as HTMLElement
```

Comment:
Tutaj trochę "podcinasz sobie skrzydła"

Ta asercja typów `as HTMLElement` ma zapobiec - "Czepianiu się" typescripta że to może być `null`

ale to prawda... i cenna infromacja - jeśli w `DOM` nie ma `#app` to wtedy `$app` będzie `null`em
Pamiętaj że `TypeScript` nie ma "rzutowania" !! 
to jedynie asercja typu.

W linii `:3` powiedziałeś do TSa: "Trust me I know what I'm doing..."
moim zdaniem warto raczej dodać tutaj `if()` jako _Type Guard_
żeby mieć `100%` pewności że operujemy na elemencie.

- czyli docenić troskę TypeScript'a :)